channels/channel_state.go

```diff
@@ -97,7 +96,11 @@ func (c channelState) Voucher() datatransfer.Voucher {
 
 // ReceivedCids returns the cids received so far on this channel
 func (c channelState) ReceivedCids() []cid.Cid {
-	return c.receivedCids
+	receivedCids, err := c.channelCIDsReader(c.ChannelID())
+	if err != nil {
+		log.Error(err)
```

Comment:
I'd consider propagating this up and letting the caller handle it, but logging it and moving on is also quite reasonable as we have "no information" in this case.